autopts/ptsprojects/boards/nrf53.py

```diff
@@ -40,13 +40,22 @@ def build_and_flash(zephyr_wd, board, debugger_snr, conf_file=None, *args):
     if conf_file and conf_file != 'default' and conf_file != 'prj.conf':
         bttester_overlay += f';{conf_file}'
 
+    host_overlay = "btstester_overlay.conf"
+    check_call(f'echo CONFIG_BT_BUF_ACL_RX_SIZE=100 > {host_overlay}'.split(),
+               cwd=tester_dir)
+    bttester_overlay += f';{host_overlay}'
+
```

Comment:
I left few comments in your PR https://github.com/zephyrproject-rtos/zephyr/pull/58328.
If you agree with me, we have to remove this
```suggestion
```

Reply:
I am fine for the fix no matter from autopts or from zephyr.